test/tests_for_sdkmr/ee2_SDKMethodRunner_test_EE2Runjob_test.py

```diff
@@ -407,10 +413,13 @@ def test_retry_job(self, rq_mock, condor_mock):
         self.check_retry_job_state(parent_job_id, retry_from_original_again)
 
         for job in [original_job, retried_job, retried_job2, retried_job3]:
+            j = Job.objects.get(id=job["job_id"])
             if job == original_job:
                 assert original_job["retry_count"] == 3
+                assert not j.retry_saved_toggle
```

Comment:
Could you check here that the original job's `retry_ids` field contains all the children?

Reply:
done

Reply:
:+1: